jbi/queue.py

```diff
@@ -240,11 +236,29 @@ def ready(self) -> list[dockerflow.checks.CheckMessage]:
                     id="queue.backend.ping",
                 )
             )
+        return results
 
+    async def check_readable(self) -> list[dockerflow.checks.CheckMessage]:
+        results = []
         try:
-            bugs_items = asyncio.run(self.retrieve())
-            for items in bugs_items.values():
-                asyncio.run(async_iter(items))
+            bugs = await self.retrieve()
+            bug_failed = 0
+            parse_success = 0
```

Comment:
Aren't they unused? 🤔

Reply:
Yeah, I meant to go back and put a `pass` in there.